src/shared/udev-util.c

```diff
@@ -88,11 +91,19 @@ int udev_parse_config_full(
                 t = resolve_name_timing_from_string(resolve_names);
                 if (t < 0)
                         log_syntax(NULL, LOG_WARNING, "/etc/udev/udev.conf", 0, r,
-                                   "failed to set parse resolve_names=%s, ignoring.", resolve_names);
+                                   "failed to parse resolve_names=%s, ignoring.", resolve_names);
                 else
                         *ret_resolve_name_timing = t;
         }
 
+        if (ret_timeout_signal && timeout_signal) {
+                r = signal_from_string(timeout_signal);
+                if (r <= 0)
+                        log_syntax(NULL, LOG_WARNING, "/etc/udev/udev.conf", 0, r, "failed to parse timeout_signal=%s, ignoring: %m", timeout_signal);
```

Comment:
This should be `< 0`.  signal_from_string() already checks the range.

Please line break here. Having this one different than all the preceding ones looks ugly.